### DISCUSSION TITLE
Assets Field "Browse" not working

### DISCUSSION THREAD

**Discussion by xaviermatos** [statamic/cms#8422] [Troubleshooting]
Whenever I try to select content from the "Browse" button using the Assets Fields, it always is slow to load and then freezes at the end of the loading process. I can only upload files directly on the field to use it. But I can't select any files that were already uploaded to my spin-up-statamic CMS Assets folder and then click the "Browse" button from the Assets Field to use the files since it will always freeze up.

**Comment by edalzell** [comment]
Gonna need some more details, Statamic version, asset container disk, any errors in the console or log?

**Reply by xaviermatos** [reply]
I'm on Statamic version 4.9.2, the asset container disk is "assets" that was on the fresh Statamic install. 

In the console I am getting a 404 (Not Found) error when I select a file from the Statamic "Browse" in the Asset Fieldtype. Another error I get is "Uncaught (in promise) Error: Request failed with status code 404 at createError, at settle, at XMLHttpRequest.ue". 

Thank you for your time.

**Reply by jackmcdade** [reply]
This is only an issue in a "Spin Up Statamic" site? If so, I have a feeling Github Workspaces doesn't really allow for arbitrary file uploads...

**Reply by xaviermatos** [reply]
I'm running the Spin-up-Statamic CMS project in local dev and getting this error.

**Reply by jackmcdade** [reply]
Okay, it might be Spin-Up in general.

**Reply by xaviermatos** [reply]
Do you have any tips to fixing the error?

**Reply by jackmcdade** [reply]
Can you open an issue on https://github.com/nystudio107/spin-up-statamic ?

**Reply by xaviermatos** [reply]
Sure, thanks for your time.
